### DISCUSSION TITLE
Incremental list to 'ask'

### DISCUSSION THREAD

**Discussion by ZhangJzhao** [scikit-optimize/scikit-optimize#1116] [Q&A]
Here is the thing, I use this library to deal with a complex issue. In my issue, the parameters need to be incremental(like 1,5,9,15,20) in range(0,200), but I didn't find function or parameter to deal with it.
Is there any strategy to deal with my case?
thanks much

**Comment by are-if** [comment]
One simple way to get incrementing parameters is:
Let (x_1, x_2, ..., x_n) be all between 0 and 1.
Define y_i =  product(x_i, x_{i+1}, ..., x_n)
Then the y_i make an increasing sequence 0 <= y_1 < y_2 < ... < y_n < 1.
You can multiply all the y's by a constant like 200, to set the range of the y's
So use x's as parameter to optimize, and then compute y's to pass to your function.
